src/analysis/normed_space/inner_product.lean

```diff
@@ -1634,37 +1636,37 @@ def submodule.orthogonal (K : submodule 𝕜 E) : submodule 𝕜 E :=
   add_mem' := λ x y hx hy u hu, by rw [inner_add_right, hx u hu, hy u hu, add_zero],
   smul_mem' := λ c x hx u hu, by rw [inner_smul_right, hx u hu, mul_zero] }
 
-/-- When a vector is in `K.orthogonal`. -/
-lemma submodule.mem_orthogonal (K : submodule 𝕜 E) (v : E) :
-  v ∈ K.orthogonal ↔ ∀ u ∈ K, ⟪u, v⟫ = 0 :=
+notation K`ᗮ`:1200 := submodule.orthogonal K
```

Comment:
Should this be `postfix`?

Reply:
Can you tell me more about `postfix`?  I can't find it discussed in TPiL.